scripts/android/after_success.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [ ! -z $TRAVIS_BRANCH ] && [ "$TRAVIS_BRANCH" != "master" ] ; then
+	export UPLOADTOOL_SUFFIX=$TRAVIS_BRANCH
+fi
+
+echo "Submitting the folloing apk for continuous build release:"
+ls -lh .../subsurface-mobile-build-arm/build/outputs/apk/*.apk
```

Comment:
three dots?
Using an absolute PATH off of ${TRAVIS_BUILD_DIR} might be better

Reply:
Ok.